awscli/customizations/codeartifact/login.py

```diff
@@ -84,13 +84,16 @@ def _run_commands(self, tool, commands, dry_run=False):
 
```

Comment:
Let's also make sure to add a changelog entry for this change so it ends up in our generated changelog. We should be able to run the `new-change` script from the root of the repository:
```
./scripts/new-change -t enhancement -c '``codeartifact login``' -d 'Include stderr output from underlying login tool when subprocess fails'
```
Then we should be able to add the json file found in the `.changes/next-release` directory.